pkg/bindings/test/exec_test.go

```diff
@@ -0,0 +1,77 @@
+package test_bindings
+
+import (
+	"time"
+
+	"github.com/containers/libpod/pkg/api/handlers"
+	"github.com/containers/libpod/pkg/bindings"
+	"github.com/containers/libpod/pkg/bindings/containers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("Podman containers exec", func() {
+	var (
+		bt *bindingTest
+		s  *gexec.Session
+	)
+
+	BeforeEach(func() {
+		bt = newBindingTest()
+		bt.RestoreImagesFromCache()
+		s = bt.startAPIService()
+		time.Sleep(1 * time.Second)
+		err := bt.NewConnection()
+		Expect(err).To(BeNil())
+	})
+
+	AfterEach(func() {
+		s.Kill()
+		//bt.cleanup()
```

Comment:
@mheon, do we need that?

Reply:
I have no idea. Copied verbatim from the other bindings tests.

Reply:
@baude, do we need the cleanup?

Reply:
yup, it has been getting copied and pasted by me too ...

Reply:
Does this mean uncomment it? Or leave it commented?

Reply:
uncomment it

Reply:
Fixed